lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -148,6 +148,23 @@ defmodule Membrane.MP4.Muxer.CMAF do
     end
   end
 
+  # public for tests 
```

Comment:
The need to make something 'public for tests' usually indicates that something is wrong with the code or the tests. Either the test is not black-box enough to call some small, private functions, or the code is too complex and should be moved/separated into another module with its own tests.